scipy/misc/__init__.py

```diff
@@ -8,33 +8,33 @@
 Various utilities that don't have another home.
 
 Note that the Python Imaging Library (PIL) is not a dependency
```

Comment:
Can you change ``Python Imaging Library (PIL)`` to ``Pillow (https://python-pillow.org/)``? PIL shouldn't be used anymore, it's abandonware.